Assets/Scripts/DynamicCamera/CameraFollowPointOfInterest.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace HarmonyQuest.DynamicCamera
+{
+    using UnityEngine;
+
+    public class CameraFollowPointOfInterest : CameraBehavior
+    {
+        public Transform targetPoint;
+        public Vector3 distanceFromPlayer;
+
+        void Awake()
+        {
+            base.Init();
+            bias = 1.5f;
+            targetAngles = new Vector3(45, 0, 0);
+        }
+
+        void Update()
+        {
+            if (targetPoint != null)
```

Comment:
We may want to add in some fail safes for enabling/disabling the point of interest camera. Stuff like clearing points of interest if the trigger is disabled, if the scene changes, etc.

Reply:
Yeah, I'd say down the road as this pull request is mainly focus on setting up the camera as minimum viable ish.